### PR TITLE
minor bug fixes and workarounds

### PR DESCRIPTION
**Pull request by f3fora**
Fix:
- unsecure-connection did not permit to change the ip and did not support ssh_config aliases
- a broken link in README.md

Add:
- an additional flag which mirrors the camera and so you can use web cam with programs which automatically flip it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +99,7 @@ $ ssh root@10.11.99.1 'chmod +x /home/root/restream'
 - `-o --output`: path of the output where the video should be recorded, as understood by `ffmpeg`; if this is `-`, the video is displayed in a new window and not recorded anywhere (default: `-`)
 - `-f --format`: when recording to an output, this option is used to force the encoding format; if this is `-`, `ffmpeg`’s auto format detection based on the file extension is used (default: `-`).
 - `-w --webcam`: record to a video4linux2 web cam device. By default the first found web cam is taken, this can be overwritten with `-o`. The video is scaled to 1280x720 to ensure compatibility with MS Teams, Skype for business and other programs which need this specific format. See [Video4Linux Loopback](#video4linux-loopback) for installation instructions.
+- `--hflip`: mirror the web cam video (`--webcam` has to be set). By default or as only choice, some programs, such as Zoom and Discord, mirror the camera. This flag restores the correct orientation.
```
Comment (rien) [inline README.md:102]: `--mirror` seems to be a more logical name for this flag. What do you think?

File: reStream.sh [hunk 2]
```diff
@@ -87,6 +93,11 @@ ssh_cmd() {
     ssh -o ConnectTimeout=1 -o PasswordAuthentication=no "root@$remarkable" "$@"
 }
 
+# SSH_CONNECTION is a variable on reMarkable => ssh '' instead of ssh ""
+remarkable_ip() {
+    ssh -o ConnectTimeout=1 -o PasswordAuthentication=no "root@$remarkable" 'echo $SSH_CONNECTION' | cut -d\  -f3
```
Comment (rien) [inline reStream.sh:98]:
```suggestion
    ssh_cmd 'echo $SSH_CONNECTION' | cut -d\  -f3
```

Comment (rien) [reply reStream.sh:98]: Just wondering, will this always return a usable IP address?

**Review by rien — COMMENTED** [review]
Apologies for waiting so long to review this PR. Can you look at comments I've made?

File: reStream.sh (hunk 2 above)
Comment (f3fora) [reply reStream.sh:98]: From `ssh` man page
> SSH_CONNECTION Identifies the client and server ends of the connection. The variable contains four space-separated values: client IP address, client port number, server IP address, and server port number.

File: README.md (hunk 1 above)
Comment (f3fora) [reply README.md:102]: Done.

File: reStream.sh (hunk 2 above)
Comment (f3fora) [reply reStream.sh]: I get a linting error with `ssh_cmd 'echo $SSH_CONNECTION' | cut -d\  -f3`, but work as expected. Tell me if for you is ok!

**Comment by f3fora** [comment]
`ssh_cmd '....'` gives some linting error, but it works as expected. Using an alias do not solve this "issue".

**Pull request merged**